### DISCUSSION TITLE
请问XLINK包含在XQUIC中么？

### DISCUSSION THREAD

**Discussion by SUOMALILI** [alibaba/xquic#64] [Q&A] [answered]
我细读了XLINK的论文觉得很有意思，我想知道XQUIC中包含XLINK么，还是说只是部分支持？

**Comment by Kulsk** [comment, marked as answer]
XLINK is an application of multipath-quic, and multipath-quic will be introduced after the draft be merged by quic-wg

**Reply by SUOMALILI** [reply]
Is that means the xquic doesn't include multipath-quic now?

**Reply by ruiqizhou** [reply]
yes, see our milestones: https://github.com/alibaba/xquic/milestone/1
